rust/chains/abacus-ethereum/abis/Outbox.abi.json

```diff
@@ -71,32 +71,32 @@
     "anonymous": false,
     "inputs": [
       {
-        "indexed": false,
```

Comment:
why is this changed?

Reply:
You probably realized before merging but for posterity these changes are bc we hadn't updated the ABIs the agents use since making some modest event name changeshere #566